### PR TITLE
Add example test for GHOST glibc vulnerability (CVE-2015-0235)

### PR DESCRIPTION
**Pull request by unixorn**
@jcasner @shantanoo please CR

### CODE REVIEW

File: examples/test_ghost.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  def testGlibcVulnerableToGhost(self):
+    """
+    glibc not vulnerable to CVE-2015-0235 (GHOST)
+    """
+    # We run this test in an external helper script because if the vulnerability
+    # exists, the test I found blows away the calling process and we don't want
+    # to miss seeing other test failures.
+    try:
+      self.assertEquals(subprocess.check_call(
+        "/path/to/your/tests/helpers/ghost-test-helper.py"), 0,
```
Comment (shantanoo) [inline examples/test_ghost.py]: Need to update the correct path of the `ghost-test-helper.py`.

Comment (unixorn) [inline examples/test_ghost.py]: This is an example test file included in the upstream source, not the tests we're running live in prod.

**Comment by unixorn** [comment]
@shantanoo @jcasner could you guys do another CR please?

**Comment by shantanoo** [comment]
LGTM.

**Pull request merged**